warden/warden_daemon/src/virtualization/vm_runner/lkvm.rs

```diff
@@ -0,0 +1,73 @@
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+use crate::managers::realm_configuration::{
+    CpuConfig, KernelConfig, MemoryConfig, NetworkConfig, RealmConfig,
+};
+
+use super::command_runner::CommandRunner;
+pub struct LkvmRunner {
+    command: Command,
+}
+
+impl LkvmRunner {
+    pub fn new(path_to_runner: PathBuf, config: &RealmConfig) -> Self {
+        let mut runner = LkvmRunner {
+            command: Command::new(path_to_runner),
+        };
+        runner.command.arg("run");
+        runner.setup_cpu(&config.cpu);
+        runner.setup_kernel(&config.kernel);
+        runner.setup_memory(&config.memory);
+        runner.setup_network(&config.network);
+        runner.control_output();
+        runner
+    }
+    pub fn configure_cca_settings(&mut self) {
+        self.command.arg("--debug");
+        self.command.arg("--irqchip=gicv3");
+        self.command.arg("--disable-sve");
```

Comment:
minor: These two aren't exactly related to CCA. The first selects the interrupt controller and the second one disables some ARM vector extension.

Reply:
Done